### ISSUE TITLE
exclude html and mermaid graph from the apk

### ISSUE THREAD

**Issue by jzeferino** [raamcosta/compose-destinations#704]
hey! 
can we add a configuration so that generated files like mermaid graph and other navigation html files are not shipped with the apk?

**Comment by raamcosta** [comment]
Hi!

Are they included? 🤔 
You mean by default if you're not setting this?

```
ksp {
    // .mmd file
    arg("compose-destinations.mermaidGraph", "$rootDir/docs")
    // .html file
    arg("compose-destinations.htmlMermaidGraph", "$rootDir/docs")
}
```

https://composedestinations.rafaelcosta.xyz/v2/defining-navgraphs#through-mermaid-graphs

**Comment by raamcosta** [comment]
Can you try the above configs?

**Comment by raamcosta** [comment]
I failed to consider this, sorry about that. In hindsight it seems so obvious 🤦 

Yes, I agree, let's disable it unless the developers provide a directory for it.

**Comment by raamcosta** [comment]
Fixed on latest v2.1.0-beta15. For older v2 versions, please set an explicit directory in the build gradle configurations so that they are generated in a directory not included in the apk builds.
